lib/hawkular/alerts/alerts_api.rb

```diff
@@ -81,12 +81,100 @@ def create_trigger(trigger, conditions = [], dampenings = [], _actions = [])
+    # @return [Array<Condition>] conditions Array of associated conditions
+    def set_group_conditions(trigger_id, trigger_mode, group_conditions_info)
+      trigger_mode = trigger_mode == :AUTORESOLVE ? 'autoresolve' : 'firing'
+      ret = http_put "triggers/groups/#{trigger_id}/conditions/#{trigger_mode}", group_conditions_info.to_h
+      conditions = []
+      ret.each { |c| conditions.push(Trigger::Condition.new(c)) }
+      conditions
+    end
+
+    # Creates a member trigger
+    # @param [GroupMemberInfo] group_member_info the group member to be added
+    # @return [Trigger] the newly created member trigger
+    def create_group_member(group_member_info)
+      ret = http_post 'triggers/groups/members', group_member_info.to_h
+      Trigger.new(ret)
+    end
+
+    # Detaches a member trigger from its group trigger
+    # @param [String] trigger_id ID of the member trigger to detach
+    # @return [Trigger] the orphan trigger
+    def orphan_member(trigger_id)
+      http_post "triggers/groups/members/#{trigger_id}/orphan", {}
+      get_single_trigger trigger_id, false
+    end
+
+    # Lists members of a group trigger
+    # @param [String] trigger_id ID of the group trigger to list members
+    # @param [boolean] orphans flag to include orphans
+    # @return [Array<Trigger>] Members found
+    def list_members(trigger_id, orphans = false)
+      ret = http_get "triggers/groups/#{trigger_id}/members?includeOrphans=#{orphans}"
+      ret.collect { |t| Trigger.new(t) }
+    end
+
+    # Creates a dampening for a group trigger
+    # @param [String] trigger_id ID of the group trigger
+    # @param [Dampening] dampening the dampening to create
+    # @return [Dampening] the newly created dampening
+    def create_group_dampening(trigger_id, dampening)
+      ret = http_post "triggers/groups/#{trigger_id}/dampenings", dampening.to_h
+      Trigger::Dampening.new(ret)
+    end
+
+    # Updates a dampening for a group trigger
+    # @param [String] trigger_id ID of the group trigger
+    # @param [String] dampening_id ID of the dampening to update
+    # @param [Dampening] dampening the dampening to update
+    # @return [Dampening] the updated dampening
+    def update_group_dampening(trigger_id, dampening_id, dampening)
+      ret = http_put "triggers/groups/#{trigger_id}/dampenings/#{dampening_id}", dampening.to_h
+      Trigger::Dampening.new(ret)
+    end
+
     # Deletes the trigger definition.
-    # @param [String] trigger_id Id of the trigger to delete
+    # @param [String] trigger_id ID of the trigger to delete
     def delete_trigger(trigger_id)
       http_delete "/triggers/#{trigger_id}"
     end
 
+    # Deletes the group trigger definition.
+    # @param [String] trigger_id ID of the group trigger to delete
+    def delete_group_trigger(trigger_id)
+      http_delete "/triggers/groups/#{trigger_id}"
+    end
+
+    # Deletes the dampening of a group trigger
```

Comment:
perhaps worth to move it couple of lines up to be next to the other dampening methods? for readability only.